examples/server/README.md

```diff
@@ -48,7 +48,7 @@ page cache before using this. See https://github.com/ggerganov/llama.cpp/issues/
 - `--path`: Path from which to serve static files. Default: disabled
 - `--api-key`: Set an api key for request authorization. By default, the server responds to every request. With an api key set, the requests must have the Authorization header set with the api key as Bearer token. May be used multiple times to enable multiple valid keys.
 - `--api-key-file`: Path to file containing api keys delimited by new lines. If set, requests must include one of the keys for access. May be used in conjunction with `--api-key`s.
-- `--embedding`: Enable embedding extraction. Default: disabled
+- `--embeddings`: Enable embedding vector output. Default: disabled
```

Comment:
Let's add: ` and the OAI compatible endpoint /v1/embeddings. Physical batch size must be carefully defined.` ?

Reply:
Sure! Thanks for the suggestion! I think it would be helpful to include `--ubatch-size` as well to make it clearer. 
Here's an example:
... Physical batch size (--ubatch-size) must be carefully defined.  

What do you think?

Reply:
Hi @phymbert @mofosyne, thanks for reviewing the request! Sorry, I just noticed your thumbs up emojis only now. I've now added phymbert's suggestion to README and have squashed the commit, and pushed the updated commit.